### PR TITLE
promote install instructions to top level menu item, enable mkdocs gh action

### PR DESCRIPTION
**Pull request by geoffcline**

### CODE REVIEW

**Comment by geoffcline** [comment]
draft new page visible here

https://geoffcline.github.io/aws-load-balancer-controller/install/installation/

**Comment by geoffcline** [comment]
ready for review @M00nF1sh :)

File: .github/workflows/mkdocs-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# GCLINE@ 23NOV2020
```
Comment (M00nF1sh) [inline .github/workflows/mkdocs-deploy.yml:1]: guess this should be amazon's copy right or leave empty :D

File: .github/workflows/mkdocs-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+jobs:
+  build:
+    name: Deploy docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+
+      - name: Deploy docs
```
Comment (M00nF1sh) [inline .github/workflows/mkdocs-deploy.yml:28]: we are using [mike](https://github.com/jimporter/mike) to deploy our multi-version docs.
I think we need to have our own actions or update `mhausenblas/mkdocs-deploy-gh-pages` to support mike.

**Comment by M00nF1sh** [comment]
btw, how about the following layout
```
Welcome
Deployment
    Installation Guide
    Configurations
    Subnet Discovery
    Pod Readiness Gate
    Upgrade
        Migrate v1 to v2
User Guide
    Ingress
        ...
    Service
        ...
    TargetGroupBinding
        ...
    Tasks
        ...
Examples
    EchoServer
```

File: .github/workflows/mkdocs-deploy.yml (hunk 1 above)
Comment (geoffcline) [reply .github/workflows/mkdocs-deploy.yml:1]: I meant that more as a "who wrote this" but I can see the confusion, I'll fix

File: .github/workflows/mkdocs-deploy.yml (hunk 2 above)
Comment (geoffcline) [reply .github/workflows/mkdocs-deploy.yml:28]: I'll take a look at how difficult that would be ... hopefully we can still do CD

**Comment by geoffcline** [comment]
ok, I can work using that layout tomorrow 👍

**Comment by geoffcline** [comment]
Ok @M00nF1sh -  I refactored based on your suggested nav.

I also removed the github action, until I can get an action for mike working. 

Thanks!
Geoffrey

**Pull request merged**